data/tabular/perovskite_db/meta.yaml

```diff
@@ -0,0 +1,155 @@
+---
+name: perovskite_db
+description: |-
+  Database of perovskite solar cells with their composition, device stacks, and performance.
+targets:
```

Comment:
Below you round to two digits. If this is the desired precision we should set it here with `significant_digits` otherwise it will get overridden by current the default = 3.

Reply:
It should, here is the code: https://github.com/OpenBioML/chemnlp/blob/main/data/text_sampling/text_sampling.py#L539

Reply:
i'll try to reproduce and open an issue in this case, but I am quite sure I got a lot of digits

Reply:
updated.